internal/mcp/agent.go

```diff
@@ -55,4 +62,26 @@ func init() {
 			return mcp_golang.NewToolResponse(mcp_golang.NewTextContent(result)), nil
 		})
 	})
+
+	register(func(c MCPContext) error {
+		return c.Server.RegisterTool("delete_agent", "this is a tool for deleting one or more Agents from the agentuity cloud platform", func(ctx context.Context, args DeleteAgentArguments) (*mcp_golang.ToolResponse, error) {
+			if resp := ensureLoggedIn(&c); resp != nil {
+				return resp, nil
+			}
+			if args.Directory != "" {
+				c.ProjectDir = args.Directory
+			}
+			if resp := ensureProject(&c); resp != nil {
+				return resp, nil
+			}
+			if len(args.AgentIds) == 0 {
+				return nil, fmt.Errorf("no agents to delete")
+			}
+			deleted, err := agent.DeleteAgents(ctx, c.Logger, c.APIURL, c.APIKey, c.Project.ProjectId, args.AgentIds)
```

Comment:
we need to remove the agent entries from the agentuity.yaml and save the file.

look at the delete command and the project package for guidance